depends/common/glm/glm.txt

```diff
@@ -0,0 +1 @@
+glm https://github.com/g-truc/glm/archive/47031aa4b7b079716e4cac496819e7f464b6a713.tar.gz
```

Comment:
Does the latest release not work?

Reply:
Has wrongly looked on https://github.com/g-truc/glm on branch names (highest 0.9.8 and not compatible) and not looked on release (there Version 0.9.9.3, which works) becomes changed.